src/main/java/com/labsynch/labseer/service/ParentAliasServiceImpl.java

```diff
@@ -19,6 +23,9 @@ public class ParentAliasServiceImpl implements ParentAliasService {
 
 	Logger logger = LoggerFactory.getLogger(ParentAliasServiceImpl.class);
 
+	@Autowired
+	private PropertiesUtilService propertiesUtilService;
+
 	@Override
 	@Transactional
 	public Parent updateParentAliases(Parent parent) {
```

Comment:
I think you're missing a validation check within this function.

Reply:
Thanks, added it.